packages/spacecat-shared-http-utils/test/auth/auth-wrapper.test.js

```diff
@@ -101,4 +117,52 @@ describe('auth wrapper', () => {
     expect(await resp.text()).to.equal('Unauthorized');
     expect(resp.status).to.equal(401);
   });
+
+  it('fetches the scope from the data layer and returns true', async () => {
+    await action(new Request('https://space.cat/', {
+      headers: { 'x-api-key': 'test' },
+    }), context);
+    const val = await context.auth.hasScopes(['scope1', 'scope2']);
+    expect(val).to.be.true;
+  });
+
+  it('returns unauthorized when api key is missing', async () => {
+    await action(new Request('https://space.cat/', {
+      headers: { 'x-api-key': 'test' },
+    }), context);
+    context.pathInfo.headers['x-api-key'] = '';
```

Comment:
Can you remove the header on line 131 instead of setting it to an empty string here?